src/modules/components/asset/assetTransfer/assetTransfer.tsx

```diff
@@ -0,0 +1,130 @@
+import classNames from 'classnames';
+import { useConfig } from 'wagmi';
+import { Avatar, AvatarIcon, IconType, NumberFormat, formatterUtils } from '../../../../core';
+import { type ICompositeAddress, type IWeb3ComponentProps } from '../../../types';
+import { AssetTransferAddress } from './assetTransferAddress';
+
+export interface IRequiredCompositeAddress extends ICompositeAddress {
+    /**
+     * Required address the transfer participants.
+     */
+    address: string;
+}
+
+export interface IAssetTransferProps extends IWeb3ComponentProps {
+    /**
+     * Address of the transaction sender.
+     */
+    sender: IRequiredCompositeAddress;
+    /**
+     * Address of the transaction recipient.
+     */
+    recipient: IRequiredCompositeAddress;
+    /**
+     * Name of the token transferred.
+     */
+    assetName: string;
+    /**
+     * Icon URL of the tranferred token.
+     */
+    assetIconSrc?: string;
+    /**
+     * Amount of tokens transferred.
+     */
+    assetAmount: number | string;
+    /**
+     * Symbol of the token transferred. Example: ETH, DAI, etc.
+     */
+    assetSymbol: string;
+    /**
+     * Price per token in fiat.
+     */
+    fiatPrice?: number | string;
+    /**
+     * Transaction hash.
+     */
+    hash: string;
+    /**
+     * Chain ID of the transaction.
+     */
+    chainId: number;
```

Comment:
The `chainId` property is already set on the `IWeb3ComponentProps` interface as optional. Let's keep this optional and use the first chain set on the wagmi config as fallback. Remember to update the tests to only pass the required properties